create_box/main.py

```diff
@@ -1,6 +1,19 @@
 def create_box(height, width, character):
-    pass
+    if height>=1 and width>=1:
+        box=""
+        for j in range(width):
+            box +=(str(character))
+        return (box + "\n")*height
 
+#create a box with outer border only
+def create_hollow_box(height, width,character):                         
+  newLine=""
+  for i in range(height):
+      if i == 0 or i == height-1:                                   #creates the first and last line of the box
+        newLine+= (width*character) + "\n" 
+      else: 
+        newLine+= (character + " "*(width-2) + character) + "\n"    #creates the hollow space in between two borders
+  return newLine
 
```

Comment:
Overall, your solution looks a bit complicated to me. Wonder if it is possible to somewhat simplify it...